README.md

```diff
@@ -47,24 +47,42 @@ Useful values:
 * **persistentStorage.enabled** can be set to *true* if you wish to save Caretta's metrics to a persistent volume *default: false*
 * **pollIntervalSeconds** can be modified to specify the polling and publishing interval of new metrics from the kernel. *default: 5*
 
+```yaml
+pollIntervalSeconds: 5.  # set metrics polling interval 
+
+tolerations:             # set any desired tolerations
+  - key: node-role.kubernetes.io/control-plane
+    operator: Exists
+    effect: NoSchedule
+    
+ victoria-metrics-single:
+  server:
+    fullnameOverride: caretta-vm
```

Comment:
why include the fullnameOverride?

Reply:
@udiz I just think we need to "visualize" the parts of the YAML you're talking about 
can you edit the way you think will do that trick?

Reply:
yup, I'll write when i'm done

Reply:
done - take a look plz

Reply:
@udiz looks great. let's merge